chart/open-feature-operator/values.yaml

```diff
@@ -148,8 +148,10 @@ managerConfig:
       ## @param managerConfig.controllerManagerConfigYaml.health.healthProbeBindAddress Sets the bind address for health probes.
       healthProbeBindAddress: :8081
     metrics:
-      ## @param managerConfig.controllerManagerConfigYaml.metrics.bindAddress Sets the bind address for metrics.
-      bindAddress: 127.0.0.1:8080
+      ## @param managerConfig.controllerManagerConfigYaml.metrics.bindAddress Sets the bind address for metrics (combined with bindPort).
+      bindAddress: 127.0.0.1
+      ## @param managerConfig.controllerManagerConfigYaml.metrics.bindPort Sets the bind port for metrics.
+      bindPort: :8080
```

Comment:
Things are a bit inconsistent here. We have a few options that are just ints, and a few that look like `:{int}`.

Should we make this consistent? @thisthat @bacherfl @odubajDT What do you guys think?

Reply:
I think in this case it might suffice to just have the `bindAddress` parameter as a string and have this set to `:8080` by default.

I think the reason we have those options that are int and ones that look like `:{int}` is because those parameters are used as `string` (e.g. `:8080`) and `int` for the related go types [here](https://github.com/open-feature/open-feature-operator/blob/339e5c8cf49c54660ed3ee70fe988345b55de999/main.go#L135) and [here](https://github.com/open-feature/open-feature-operator/blob/339e5c8cf49c54660ed3ee70fe988345b55de999/main.go#L146) (the webhook server port seems to be hard coded as well btw)

So i think we can probably keep it this way

Reply:
I'm unclear on what changes you want made here?

Re-write so that bindAddress is simply `:8080` and remove the `127.0.0.1`? Wouldn't that hurt flexibility for if a user really wanted another address?

The webhook server port can be addressed in a separate PR.

Reply:
I would suggest to just use `bindAddress` and set that to `:8080` by default. This makes it a bit clearer that this value directly relates to the `--metrics-bind-address` flag of the operator. The value provided here does not need to be only a port, but can contain the address as well (see https://pkg.go.dev/net#Listen), so this should not reduce flexibility.
regarding the webhook server port, I agree that - if necessary - this can be done in a separate PR

Reply:
So I've worked on this a little further and concluded:
- Actually we should do webhook port with this one as both ports are relevant and could cause a port conflict when `hostNetwork` is true. I've added it to the deployment.
- I've also added the bindPort to the deployment as well, but this means we can't have a `:` in front of the port number and have to introduce that in the Helm templating, as it's required to be a number (I tested it!).